python/test/kv/kvTest.py

```diff
@@ -95,6 +97,25 @@ def testCopy(mode):
         kv.destroy()
     return True
 
+# Test multiprocessing by using subprocess
+def testMultiproc(mode):
+    if mode == 'sharemem':
+        pypath = sys.executable
+        p1 = sp.Popen([pypath, "./multiproc/test1.py"])
+        p2 = sp.Popen([pypath, "./multiproc/test2.py"])
+        p3 = sp.Popen([pypath, "./multiproc/test3.py"])
+        p3.wait()
+        p2.wait()
+        p1.wait()
+        if p3.returncode != 0 or p2.returncode != 0:
+            return False
+        memory = posix_ipc.SharedMemory("share")
+        mapfile = mmap.mmap(memory.fd, memory.size)
+        memory.close_fd()
+        print(mapfile[:1000].rstrip(b'\x00'))
```

Comment:
remove this before committing. It isn't useful output to the user. You can comment it out if it will be useful for debugging later.